docs/installation/binary_installation.md

```diff
@@ -0,0 +1,82 @@
+# Binary Installation
+
+The packages for analyzing trace data with CARET can also be installed in binary.
+
+<prettier-ignore-start>
+!!! Note
+    Only the packages for analyzing trace data can be installed by binary installation.<br>
+    The binary does not include the packages for recording trace data.
+<prettier-ignore-end>
+
+The table of packages installable via binary installation is following.
+
+| packages name                                                                 | packages description                                                          | binary install |
+| ----------------------------------------------------------------------------- | ----------------------------------------------------------------------------- | :------------: |
+| [caret](https://github.com/tier4/caret)                                       | Meta-repository                                                               |       ×        |
+| [caret_trace](https://github.com/tier4/caret_trace/tree/main/CARET_trace)     | Define tracepoints added by function hooking                                  |       ×        |
+| [caret_msgs](https://github.com/tier4/caret_trace/tree/main/caret_msgs)       | Message type defined by caret                                                 |       ○        |
+| [caret_analyze](https://github.com/tier4/caret_analyze)                       | Library for scripts to analyze and visualize data                             |       ○        |
+| [caret_analyze_cpp_impl](https://github.com/tier4/caret_analyze_cpp_impl.git) | Efficient helper functions to analyze trace data written in C++               |       ○        |
+| [ros2caret](https://github.com/tier4/ros2caret.git)                           | CLI commands like `ros2 caret`                                                |       ○        |
+| [caret_doc](https://github.com/tier4/caret_doc)                               | Documentation                                                                 |       ×        |
+| [caret_demos](https://github.com/tier4/caret_demos)                           | Demo programs for CARET                                                       |       ×        |
+| [rclcpp](https://github.com/tier4/rclcpp/tree/v0.3.0)                         | The forked rclcpp including CARET-dedicated tracepoints                       |       ×        |
+| [ros2_tracing](https://github.com/tier4/ros2_tracing/tree/v0.3.0)             | The forked `ros2_tracing` including definition of CARET-dedicated tracepoints |       ×        |
+
```

Comment:
Please remove caret and caret_doc. They are not ROS packages.
I think caret_demos should be removed. It has nothing to do with the functionality provided by caret.

Reply:
Fixed in https://github.com/tier4/caret_doc/pull/335/commits/f123f0bce7bf4f20e2f9dcf33ea016664c2fb473.

Reply:
Fixed in https://github.com/tier4/caret_doc/pull/335/commits/f123f0bce7bf4f20e2f9dcf33ea016664c2fb473.